man/org.freedesktop.portable1.xml

```diff
@@ -234,11 +234,16 @@ node /org/freedesktop/portable1 {
         <listitem><para>copy</para></listitem>
 
         <listitem><para>symlink</para></listitem>
+
+        <listitem><para>mixed</para></listitem>
       </itemizedlist>
-      This method returns the list of changes applied to the system (for example, which unit was
-      added and is now available as a system service). Each change is represented as a triplet of
-      strings: the type of change applied, the path on which it was applied, and the source
-      (if any). The type of change applied will be one of the following possible values:
+      <varname>copy</varname> will prefer copying, <varname>symlink</varname> will prefer linking if
```

Comment:
"prefer copying"? it sounds as if it would then still sometimes symlink. But it won't, will it? If you specify "copy", then it will *always* copy, no?